canister/src/api/get_utxos.rs

```diff
@@ -114,6 +113,15 @@ fn get_utxos_internal(
         // A page was provided in the request, so we should use it as a basis
         // to compute the next chunk of UTXOs to be returned.
         Some(page) => {
+            if page.len() != 72 {
+                return Err(GetUtxosError::MalformedPage {
+                    err: format!(
+                        "Could not parse page, page size {}, expected size {}.",
+                        page.len(),
+                        72
+                    ),
+                });
+            }
```

Comment:
I'd move this logic inside `Page::from_bytes` as that's where it logically belongs. I'd also define a constant for the expected page length and use that (i.e. `const EXPECTED_PAGE_LENGTH: usize = 72`.